crates/forge_permission_resolver/src/permissions_resolver.rs

```diff
@@ -199,12 +214,25 @@ fn get_request_type(
 }
 
 fn get_scopes(endpoint_data: &RequestDetails) -> Vec<String> {
-    endpoint_data
+    let mut scopes = endpoint_data
         .permission
         .iter()
         .flat_map(|data| &*data.scopes)
         .cloned()
-        .collect()
+        .collect::<Vec<_>>();
+
+    if scopes.is_empty() {
+        // For Jira Software if the initial scopes are empty, try the scopes from the security field
+        scopes.extend(
```

Comment:
Are there any cases where the `security` field has additional scopes with a nonempty `permission` field?